src/Raven.Server/Utils/ThreadsUsage.cs

```diff
@@ -98,6 +100,10 @@ public ThreadsInfo Calculate()
                     {
                         // thread has exited
                     }
+                    catch (Win32Exception e) when (e.Message.Contains("Access is denied", StringComparison.OrdinalIgnoreCase))
```

Comment:
That is a bad idea. Windows messages are localized.
What is the error code here?

Reply:
done